scripts/run_frontend_tests.py

```diff
@@ -53,6 +53,10 @@
     '--skip_install',
     help='optional; if specified, skips installing dependencies',
     action='store_true')
+_PARSER.add_argument(
+    '--debug_mode',
```

Comment:
Per offline discussion, use `--verbose` instead to more clearly communicate the argument's effect

Reply:
Done!